gef-ui/server/controllers/application-submission/index.js

```diff
@@ -16,6 +16,8 @@ const getApplicationSubmission = async (req, res) => {
     dealId,
     submissionType,
     maxCommentLength,
+    isAutomaticCover: application.submissionType,
```

Comment:
`isAutomaticCover` should it be renamed as application.submissionType will return MIA,AIN or MIN.
Perhpas a helper function like `isNotice` might be appropriate?

Reply:
can this be for a future PR?

Reply:
Might be better to do it now since you are already at it please.

Reply:
have fixed this by removing isAutomaticCover and using submissionType with capitalize pipe.  It works.  Is that fine with you @abhi-markan @ttbarnes ?